ethereum/api/src/integration-test/java/org/hyperledger/besu/ethereum/api/jsonrpc/methods/DebugStandardTraceBlockToFileIntegrationTest.java

```diff
@@ -0,0 +1,225 @@
+import org.hyperledger.besu.testutil.BlockTestUtil;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.io.Resources;
+import com.ibm.icu.impl.Assert;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.CleanupMode;
+import org.junit.jupiter.api.io.TempDir;
+
+public class DebugStandardTraceBlockToFileIntegrationTest {
+  private static final String RPC_ENDPOINT = "debug_standardTraceBlockToFile";
+  private static JsonRpcTestMethodsFactory blockchain;
+  private JsonRpcMethod method;
+
+  @TempDir(cleanup = CleanupMode.ON_SUCCESS)
+  private static Path dataDir;
+
+  @BeforeAll
+  public static void setUpOnce() throws Exception {
+    // experimental tracers are meant to be enabled in tests
+    DebugTraceTransactionStepFactory.enableExtraTracers = true;
+
+    final String genesisJson =
+        Resources.toString(BlockTestUtil.getTestGenesisUrl(), StandardCharsets.UTF_8);
+
+    blockchain =
+        new JsonRpcTestMethodsFactory(
+            new BlockchainImporter(BlockTestUtil.getTestBlockchainUrl(), genesisJson), dataDir);
+  }
+
+  @BeforeEach
+  public void setUp() {
+    final Map<String, JsonRpcMethod> methods = blockchain.methods();
+    method = methods.get(RPC_ENDPOINT);
+  }
+
+  @Test
+  public void defaultFieldsMemory() throws IOException {
+    final Hash blockHash =
+        Hash.fromHexString("0x10aaf14a53caf27552325374429d3558398a36d3682ede6603c2c6511896e9f9");
+    final Object[] params = new Object[] {blockHash};
+    final JsonRpcRequestContext request =
+        new JsonRpcRequestContext(new JsonRpcRequest("2.0", RPC_ENDPOINT, params));
+
+    final JsonRpcResponse response = method.response(request);
+    assertThat(response.getType()).isEqualTo(RpcResponseType.SUCCESS);
+    List<?> files = (List<?>) ((JsonRpcSuccessResponse) response).getResult();
+
+    ObjectMapper jsonMapper = new ObjectMapper();
+    List<JsonNode> json =
+        Files.readAllLines(Path.of((String) files.getFirst())).stream()
+            .map(
+                line -> {
+                  try {
+                    return jsonMapper.readTree(line);
+                  } catch (JsonProcessingException e) {
+                    Assert.fail("encountered invalid json from RPC response");
+                    throw new RuntimeException(e);
+                  }
+                })
+            .toList();
+
+    assertThat(json).anyMatch(node -> node.has("memory"));
+    assertThat(json).anyMatch(node -> node.has("stack"));
+  }
+
+  @Test
+  public void defaultFieldsStorage() throws IOException {
```

Comment:
what's the difference between this test and `defaultFieldsMemory` - they are doing the same steps, just asserting different aspects of the output?